_scripts/prs.py

```diff
@@ -0,0 +1,81 @@
+"""
+Summarize PRs for fortran-lang within a certain month.
+Requires PyGithub.
+"""
+import datetime
+import os
+from typing import Optional
+
+import github
+
+
+def get_prs(
+    repo: github.Repository.Repository, 
+    t_a: datetime.datetime,
+    t_b: datetime.datetime,
+):    
+    merged = []  # PRs merged during the month
+    for pr in repo.get_pulls("closed", sort="updated", direction="desc"):  # most recently merged first
+        if pr.merged:
+            if t_a <= pr.merged_at < t_b:  # could check `.month` instead...
+                merged.append(pr)
+            if pr.merged_at < t_a:
+                break
+            
+    wip = []  # WIP PRs (not merged, still open at this time)
+    for pr in repo.get_pulls("open"):
+        if pr.created_at < t_b: # and pr.updated_at >= t_a:
+            wip.append(pr)
+    
+    return merged, wip
+
+
+def main(sdate: Optional[str], token: Optional[str]):
+    # Time range: [t_a, t_b)
+    if sdate is None:
+        now = datetime.datetime.now()
+        t_a = datetime.datetime(now.year, now.month, 1)
+    else:
+        t_a = datetime.datetime.strptime(sdate, "%Y-%m")
+    t_b = datetime.datetime(t_a.year, t_a.month+1, 1)
+
+    # GitHub connection
+    if token is None:
+        token = os.environ.get("GITHUB_TOKEN")
+    if token is None:
+        print("token not set. GitHub API may complain")
+    g = github.Github(token, per_page=100)
+    org = g.get_organization("fortran-lang")
+
+    for reponame in ["stdlib", "fpm"]:
```

Comment:
The webpage is usually in the newsletter as well.
```suggestion
    for reponame in ["fortran-lang.org", "stdlib", "fpm"]:
```
There are plenty of other repositories, like `setup-fpm`, `stdlib-cmake-example`, `fpm-haskell` in the @fortran-lang namespace with low activity I would usually associate with the parent project if something is going on there.

Reply:
Hmm so would it be more helpful for their PRs to be in the list for the parent project or have their own separate lists?

Reply:
Given that those projects are usually on a rather low activity level, I don't think they warrant their own section. In the past we announced those projects in the section of their respective parent project, I'm not sure if we picked up all setup-fpm updates in the past in the fpm section, actually.

Reply:
Ok, I added this capability in https://github.com/fortran-lang/fortran-lang.org/pull/261/commits/7de8b764e4e16a711e2bb1516720aa1f2e751d70, let me know what you think. You can see it working with `--month 2021-03` for example